chainerx_cc/chainerx/routines/linalg.cc

```diff
@@ -325,4 +326,63 @@ Array PseudoInverse(const Array& a, float rcond) {
     return out;
 }
 
+std::tuple<Array, Array> Eigh(const Array& a, const std::string& uplo) {
```

Comment:
As `uplo` is a binary option, how about simply using a bool, in C++? Although that might compromise readability, it should be faster. We could name it `use_a_lower`, `is_a_lower`, ...?

Reply:
It may be faster or may not. However this char/string flag is used in LAPACK and NumPy, I think it is worth keeping C++ interface consistent with those libraries.

Reply:
How about using a char instead of string?